adapters/info.go

```diff
@@ -206,6 +206,7 @@ type BidderInfo struct {
 	Capabilities            *CapabilitiesInfo `yaml:"capabilities" json:"capabilities"`
 	AliasOf                 string            `json:"aliasOf,omitempty"`
 	ModifyingVastXmlAllowed bool              `yaml:"modifyingVastXmlAllowed" json:"-" xml:"-"`
+	DebugInfo               *config.DebugInfo `yaml:"debug,omitempty" json:"debug,omitempty"`
```

Comment:
Would calling it `Debug` instead of `DebugInfo` cause a naming conflict?

Reply:
Renamed, no conflicts